hdinsight-genomics-adam-latest-build/azuredeploy.json

```diff
@@ -0,0 +1,227 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
```

Comment:
similar comments as above apply to this template as well. Also, please remove "latest build" from the folder name